From deb3b0ab6a97f4584fe37b3ef708dcad59ceddde Mon Sep 17 00:00:00 2001
From: Alexander Larsson <alexl@redhat.com>
Date: Fri, 12 Mar 2010 16:23:42 +0100
Subject: [PATCH 1/9] Add CONVERT_0565_TO_8888 macro

This lets us simplify some fast paths since we get a consistent
naming that always has 8888 and gets some value for alpha.
---
 pixman/pixman-private.h |    2 ++
 1 files changed, 2 insertions(+), 0 deletions(-)

diff --git a/pixman/pixman-private.h b/pixman/pixman-private.h
index 9dcdca7..d0bec39 100644
--- a/pixman/pixman-private.h
+++ b/pixman/pixman-private.h
@@ -704,6 +704,8 @@ pixman_region16_copy_from_region32 (pixman_region16_t *dst,
      ((((s) << 5) & 0xfc00) | (((s) >> 1) & 0x300)) |			\
      ((((s) << 8) & 0xf80000) | (((s) << 3) & 0x70000)))
 
+#define CONVERT_0565_TO_8888(s) (CONVERT_0565_TO_0888(s) | 0xff000000)
+
 #define PIXMAN_FORMAT_IS_WIDE(f)					\
     (PIXMAN_FORMAT_A (f) > 8 ||						\
      PIXMAN_FORMAT_R (f) > 8 ||						\
-- 
1.6.6.1

